packages/components/src/Api/Subscription.purs

```diff
@@ -37,8 +37,10 @@ type Subscription =
   , paused                :: Nullable (Array PausedSubscription)
   , deliveryAddress       :: Nullable DeliveryAddress
   , pendingAddressChanges :: Nullable (Array PendingAddressChange)
+  , paymentMethod         :: SubscriptionPaymentMethod
   }
 
+newtype SubscriptionPaymentMethod = SubscriptionPaymentMethod String
```

Comment:
Instead of a `newtype`, let's make this into a sum type, because when we'll start adding features concerning the payments, it'll come handy.

```purescript
data SubscriptionPaymentMethod
  = PaperInvoice
  | CreditCard
  | NetBank
  ...
```